### ISSUE TITLE
macOS Ventura - This repository is configured for Git LFS but 'git-lfs' was not found on your path.

### ISSUE THREAD

**Issue by colinmacleod** [GitFiend/Support#183]
When I try to push staged changes to a GIT LFS repository, I get this error:

`This repository is configured for Git LFS but 'git-lfs' was not found on your path. If you no longer wish to use Git LFS, remove this hook by deleting the 'pre-push' file in the hooks directory (set by 'core.hookspath'; usually '.git/hooks').

error: failed to push some refs to 'ssh://git.bitquirky.io:22070/colinmacleod/luminous-threads.git'`

git-lfs is on my path in terminal; I'm running zsh and it was installed via homebrew.

How do I point Gitfiend to the homebrew bin folder?

**Comment by GitFiend** [comment]
Thanks for the report. Sorry, the path stuff is currently hard-coded. I think I'm going to have to replicate this and come up with a fix.

**Comment by colinmacleod** [comment]
Let me know if I can help replicate.
It's a fairly clean install on a mac-mini M1.

**Comment by GitFiend** [comment]
I noticed Tauri has a path env fixer util that does more than my own and mentions zsh ([fix-path-env-rs](https://github.com/tauri-apps/fix-path-env-rs/tree/dev)). It may be a stab in the dark, but I've created a build that uses this fix: https://github.com/GitFiend/Support/releases/tag/0.43.0

Otherwise, If that doesn't work I'll need to set up a vm, git hook and investigate.

**Comment by colinmacleod** [comment]
Yes, that seems to do the trick. Nice one - and well done! Thank you very much!

**Comment by GitFiend** [comment]
Nice. This will probably fix some other related stuff as well, so thanks for bringing it up.
